### PR TITLE
Update docker docu with new docker image tags

### PR DESCRIPTION
**Pull request by cniweb**
Update from README.md at openhab/openhab-docker #90

Signed-off-by: Christian Häussler <c-n-i@web.de> (github: @cniweb)

### CODE REVIEW

File: installation/docker.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ mkdir /opt/openhab
 mkdir /opt/openhab/conf
 mkdir /opt/openhab/userdata
 mkdir /opt/openhab/addons
-chown -R openhab:openhab /opt/openhab
+chown 9001.9001 /opt/openhab -R
```
Comment (ThomDietrich) [inline installation/docker.md]: The format `chown -R openhab:openhab /opt/openhab` is more common, why the change?

**Comment by cniweb** [comment]
I confirm with you and I switch back

**Comment by ThomDietrich** [comment]
Everything looks great. Thank you!
